### DISCUSSION TITLE
Proposal to pay for work done in January 2024

### DISCUSSION THREAD

**Discussion by IgorShadurin** [fairDataSociety/Fairdrive-DAO#11] [Proposals]
https://xdai.colony.io/colony/fairdrive/decisions/tx/0x7e543a078d99a905702430884f3170578b413b83112bbc54c211250b8c6cef28
https://hackmd.io/@-Uo3Z00KQnaG-H_ZMAQx1Q/HJuZGD-1A

**Comment by tfius** [comment]
Why is still not done ?
